zerver/webhooks/dialogflow/doc.md

```diff
@@ -0,0 +1,23 @@
+Know the results of your queries on your dialogflow app by PM in Zulip !
+
+{!create-bot-construct-url.md!}
+
+url structure should be like this:
+
+`{{api_url}}?api_key=BOT'S_API_KEY&email=foo@example.com`
+
+Modify the parameters of the url above where `api_key` is api key of your Zulip bot \
+and `email` is your zulip email.
+
+* Go to `Fulfillment` setting of your dialogflow app and `enable` Webhooks \
+and provide there the url the webhooks should be sent to.
+
+![](/static/images/integrations/dialogflow/001.png)
+
+* Then go to `Intents` and at bottom check the box named `Use webhook` under heading `Fulfillment`.
+
```

Comment:
Change this to `Then go to the **Intents** and at the bottom, check the box **Use webhook** under **Fulfillment**.`